src/main/java/com/onevote/command/validation/OptionValidation.java

```diff
@@ -0,0 +1,48 @@
+package com.onevote.command.validation;
+
+import com.onevote.Option;
+import com.onevote.Vote;
+import com.onevote.exception.OneVoteRuntimeException;
+import java.util.Date;
+import org.springframework.util.StringUtils;
+
+public class OptionValidation {
+  private OptionValidation() {
+    throw new AssertionError();
+  }
+
+  public static void validateCreateOption(Option option, Vote vote, String userName) throws OneVoteRuntimeException {
+    // Check Option name is falsy
+    if (StringUtils.isEmpty(option.getName())) {
+      throw new OneVoteRuntimeException("option name is empty or null");
+    }
+
+    // Check name max length
+    if (option.getName().length() >= 200) {
+      throw new OneVoteRuntimeException("option name exceeds the maximum of 200 characters");
+    }
+
+    // Check option uniqueness
+    for (Option op : vote.getOptions()) {
+      if (op.getName().equals(option.getName())) {
```

Comment:
the logic can be improved here. 
if we find a matched option, but it is deactived, then we can active it, right?